### PR TITLE
Added good-hipchat support.

### LINKED ISSUE #14: good-hipchat

### PR DESCRIPTION
**Pull request by dawnerd**
Fixes #14

### CODE REVIEW

**Comment by jgallen23** [comment]
Will it just do nothing if we don't set a token?

**Comment by dawnerd** [comment]
With debug:true it logs it, but with no token and debug:false it wont auth with hipchat.

**Comment by jgallen23** [comment]
and it doesn't try, right?

I think debug should be false by default

**Comment by dawnerd** [comment]
By default it isn't loaded since there's no reported defined in the conf/default.yaml

**Comment by jgallen23** [comment]
AHH!  thought it was conf/default.yaml

File: example/conf/default.yaml [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+logging:
+  reporters:
+    hipchat:
+      - 
+        auth_token: ''
+        room: ''
+        roomToken: ''
+        debug: true
+        notify: false
+        events:
```
Comment (jgallen23) [inline example/conf/default.yaml]: I think these should be in default, so we just have to set bare minimum to get up and running

Comment (jgallen23) [inline example/conf/default.yaml]: thoughts?

Comment (dawnerd) [inline example/conf/default.yaml]: I think that'd be fine, problem is the way its loaded. If we have the hipchat reporter there the module will be loaded. Maybe have an enable flag?

Comment (jgallen23) [inline example/conf/default.yaml]: I like the enable flag.  I have that on the plugins side, make sense to do on the logging side

**Pull request merged**